int32 sensor_type
---
string pipeline
int32 error